style.css

```diff
@@ -0,0 +1,771 @@
+  line-height: 24px;
+  margin-left: 20px;
+  margin-top: 30px;
+}
+
+@media screen and (min-width: 768px) {
+  .black-navbar {
+    position: fixed;
+    display: flex;
+    flex-direction: row;
+    background-color: #3e3e3e;
+    height: 30px;
+    padding-left: 60vw;
+    width: 100%;
+    align-items: center;
+  }
+
+  .black-navbar .social-icons-container {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-top: 5px;
+    padding-bottom: 5px;
+  }
+
+  .links-container ul li {
+    margin-left: 45px;
+    cursor: pointer;
+  }
+
+  .black-navbar .social-icons-container li {
+    margin-right: 25px;
+  }
+
+  .links-container ul li a {
+    font-family: "Lato", sans-serif;
+    font-size: 14px;
+    color: #868686;
+  }
+
+  .black-navbar .social-icons-container li a {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 14px;
+    color: #fff;
+  }
+
+  .navbar {
+    margin-top: 30px;
+    background-color: white;
+    padding-left: 150px;
+    padding-right: 150px;
+    height: 130px;
+  }
+
+  #toggle-icon {
+    display: none;
+  }
+
+  .navbar .logo-container {
+    margin-left: 15px;
+    display: block;
+  }
+
+  footer .footer-logo-container img {
+    width: 180px;
+  }
+
+  .navbar .logo-container img {
+    width: 100px;
+  }
+
+  .links-container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .links-container ul {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-top: 40px;
+    padding-right: 55px;
+  }
+
+  .links-container .cc-button {
+    padding: 5px 25px;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 14px;
+    line-height: 17px;
+    color: #ec4327;
+    border: 5px solid #ec4327;
+    background-color: #fff;
+    width: 150px;
+    height: 50px;
+    cursor: pointer;
+    margin-top: 30px;
+  }
+
+  .header,
+  .main-program,
+  .professors-list {
+    padding-left: 150px;
+    padding-right: 150px;
+  }
+
+  .main-program .programs-list {
+    display: grid;
+    grid-template-columns: repeat(5, 175px);
+  }
+
+  .main-program .programs-list .program-container {
+    display: flex;
+    flex-direction: column;
+    margin-left: 20px;
+    min-height: 200px;
+    align-items: center;
+    padding: 10px 10px;
+  }
+
+  .partners .partner-icons-list li img {
+    width: 150px;
+  }
+
+  .main-program .programs-list .program-container img {
+    width: 35px;
+  }
+
+  .main-program .programs-list .program-container .program-title {
+    margin-top: 15px;
+    text-align: justify;
+    max-width: 100px;
+    height: 20px;
+  }
+
+  .main-program .programs-list .program-container .program-desc {
+    margin-top: 25px;
+    text-align: start;
+    max-width: 120px;
+  }
+
+  .featured-professors .professors-list {
+    align-self: center;
+    display: grid;
+    grid-template-columns: repeat(2, 660px);
+  }
```

Comment:
- [ ] I noticed that the speakers section is not properly aligned according to the given template, though it wasn't like this before; perhaps you mistakenly made the mistake in the process of making the changes required earlier. Kindly work on it. 👍 

## Your Design
<img width="945" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225860116-973d8e30-f481-4c74-9af6-dd27a6fdcac6.png">